mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/components/SlippageTolerance.tsx

```diff
@@ -1,124 +1,211 @@
 import { ThemedIcon, ThemedText, ThemedView } from '@components/themed'
 import { View } from '@components'
-import { BottomSheetSlippageTolerance } from './BottomSheetSlippageTolerance'
-import { BottomSheetNavScreen } from '@components/BottomSheetWithNav'
+import { BottomSheetInfo } from '@components/BottomSheetInfo'
+import { WalletTextInput } from '@components/WalletTextInput'
+import { debounce } from 'lodash'
 
-const SLIPPAGE_MODAL_NAME = 'SlippageTolerance'
 const percentageList = ['1', '3', '5', '10', '20']
 
 interface SlippageToleranceProps {
   slippage: BigNumber
+  slippageError?: SlippageError
+  setSlippageError: (error?: SlippageError) => void
   setSlippage: (slippage: BigNumber) => void
-  setIsSelectorOpen: (val: boolean) => void
-  setBottomSheetScreen: (screens: BottomSheetNavScreen[]) => void
-  onPress: () => void
-  onCloseButtonPress: () => void
 }
 
-export function SlippageTolerance ({ slippage, setSlippage, setIsSelectorOpen, setBottomSheetScreen, onPress, onCloseButtonPress }: SlippageToleranceProps): JSX.Element {
-  const { dismiss } = useBottomSheetModal()
+export function SlippageTolerance ({ slippage, setSlippage, slippageError, setSlippageError }: SlippageToleranceProps): JSX.Element {
   const [isCustomSlippage, setIsCustomSlippage] = useState(false)
-  const onSubmitSlippage = (val: BigNumber, isCustom: boolean): void => {
-    setIsCustomSlippage(isCustom)
-    setSlippage(val)
-    setIsSelectorOpen(false)
-    dismiss(SLIPPAGE_MODAL_NAME)
+  const slippageTolerance = {
+    title: 'Slippage tolerance',
+    message: 'Slippages are rate changes that occur within an order transaction. Choose how much of this slippage are you willing to accept.'
   }
 
-  const setSlippageToleranceToBottomSheet = useCallback(() => {
-    setBottomSheetScreen([
-      {
-        stackScreenName: 'BottomSheetSlippageTolerance',
-        component: BottomSheetSlippageTolerance({
-          slippage,
-          isCustomSlippage,
-          onSubmitSlippage,
-          onCloseButtonPress
-        }),
-        option: {
-          header: () => null
-        }
-      }])
-  }, [slippage, isCustomSlippage])
-
   useEffect(() => {
     setIsCustomSlippage(!percentageList.includes(new BigNumber(slippage).toString()))
-  }, [slippage])
+  }, [])
 
   return (
     <ThemedView
-      dark={tailwind('bg-gray-800 border-b border-gray-700')}
-      light={tailwind('bg-white border-b border-gray-200')}
       style={tailwind('flex-row items-center w-full')}
     >
-      <View style={tailwind('w-full')}>
-        <TouchableOpacity
-          onPress={() => {
-            setIsSelectorOpen(true)
-            setSlippageToleranceToBottomSheet()
-            onPress()
-          }}
-          style={tailwind('p-4 pr-2')}
-          testID='slippage_select'
+      <View style={tailwind('px-4 mt-5 w-full')}>
+        <View
+          testID='slippage_tolerance_heading'
+          style={tailwind(['flex-row mb-3 items-center justify-between'])}
```

Comment:
Let's move the title closer to the selector
```suggestion
          style={tailwind(['flex-row items-center justify-between'])}
```